### PR TITLE
Update ja/docs/concepts/workloads/pods/_index.md

### LINKED ISSUE #26900: ja: Make docs/concepts/workloads/pods/_index.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/concepts/workloads/pods/_index.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/concepts/workloads/pods/_index.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/workloads/pods/_index.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/workloads/pods/_index.md

diff between translated and v1.19
https://gist.github.com/995a155ef0ec032a59728a84ead29ee6

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by shuuji3**
✅  Resolves #26900.

### CODE REVIEW

File: content/ja/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -1,4 +1,160 @@
...
+title: Pod
+content_type: concept
 weight: 10
+no_list: true
+card:
+  name: concepts
+  weight: 60
 ---
+
+*Pod*は、Kubernetes内で作成・管理できるコンピューティングの最小のデプロイ可能なユニットです。
```
Comment (shuuji3) [inline content/ja/docs/concepts/workloads/pods/_index.md:11]: https://github.com/kubernetes/website/blob/release-1.19/content/en/docs/concepts/workloads/pods/_index.md#L17

原文には "(as in a pod of whales or pea pod)" という表現([クラゲの群れ](https://images.app.goo.gl/N77PGvzZuT8GxpSS9)や[えんどう豆のさや](https://www.google.com/search?q=pea+pod)という意味らしいです)があって面白いと思ったのですが、日本語にすると分かりづらいと思ったので省略してしまいました。

File: content/ja/docs/concepts/workloads/pods/_index.md [hunk 2]
```diff
@@ -1,4 +1,160 @@
 ---
-title: "Pod"
+title: Pod
```
Comment (inductor) [inline content/ja/docs/concepts/workloads/pods/_index.md:2]: ここ変えなくてよいかと

**Review by inductor — CHANGES_REQUESTED** [review]

File: content/ja/docs/concepts/workloads/pods/_index.md (hunk 2 above)
Comment (shuuji3) [reply content/ja/docs/concepts/workloads/pods/_index.md:2]: たしかにそうですね。#27626 もそのままにするべきでした。こちらは修正します。

File: content/ja/docs/concepts/workloads/pods/_index.md (hunk 1 above)
Comment (atoato88) [reply content/ja/docs/concepts/workloads/pods/_index.md:11]: As refer other localization(ex: [zh](https://v1-19.docs.kubernetes.io/zh/docs/concepts/workloads/pods/), [ko](https://v1-19.docs.kubernetes.io/ko/docs/concepts/workloads/pods/)), zh translate the sentence by localized sentence only.
ko writes both of localized sentence and original words.
IMHO, I would like to select ko style.

Comment (shuuji3) [reply content/ja/docs/concepts/workloads/pods/_index.md:11]: I checked Korean localization, and I agree with you to follow that style as you suggest. I'll update the translation to include "(as in a pod of whales or pea pod)".

Comment (shuuji3) [reply content/ja/docs/concepts/workloads/pods/_index.md]: I added a missing translation of "(as in a pod of whales or pea pod)". I added it to the end to make Japanese text more readable.

Comment (shuuji3) [reply content/ja/docs/concepts/workloads/pods/_index.md]: FYI: I found an [interesting conversion](https://twitter.com/thockin/status/1060301866070814720) related to "pod of whales".

Comment (atoato88) [reply content/ja/docs/concepts/workloads/pods/_index.md]: Wow!
Thank you to let me know that! :whale: :whale: :whale:

**Review by atoato88 — COMMENTED** [review]
Here are some nits.

File: content/ja/docs/concepts/workloads/pods/_index.md (hunk 1 above)
Comment (shuuji3) [reply content/ja/docs/concepts/workloads/pods/_index.md]: Welcome! I'd like to know about other conversations like this.😃 Maybe someone has already created a tweet collection about Kubernetes.

**Pull request merged**